### PR TITLE
home page for quiz app created

### PR DESCRIPTION
**Pull request by aishwarya-bm**
Home page with banner and title
Category list

### CODE REVIEW

File: src/pages/question/question.css [hunk 1]
```diff
@@ -18,13 +18,13 @@
 }
 
 .option-item {
-    color: white;
+    /* color: white; */
```
Comment (srejitk) [inline src/pages/question/question.css]: You can remove this comment if not needed

**Review by srejitk — APPROVED** [review]
Clean code, found your use of Scroll into view interesting. Looks good. No other changes suggested.

File: src/pages/question/question.css (hunk 1 above)
Comment (aishwarya-bm) [reply src/pages/question/question.css]: Done, thanks!

File: src/pages/result/result.css [hunk 2]
```diff
@@ -12,10 +12,10 @@
     width: fit-content;
     align-self: center;
     font-size: 1.25rem;
-    background-color: var(--contrast-color);
-    color: var(--black-color);
+    /* background-color: var(--contrast-color); */
+    /* color: var(--black-color); */
```
Comment (sadathshariff) [inline src/pages/result/result.css]: remove the commented code here

**Review by sadathshariff — APPROVED** [review]
Overall the code looks clean and structured, learnt about useRef

File: src/pages/result/result.css (hunk 2 above)
Comment (aishwarya-bm) [reply src/pages/result/result.css]: Done, thanks!

**Pull request merged**